numpy/core/_add_newdocs.py

```diff
@@ -3953,13 +3953,23 @@
 
     Examples
     --------
-    For a 1D array, ``a.tolist()`` is almost the same as ``list(a)``:
+    For a 1D array, ``a.tolist()`` is almost the same as ``list(a)``, except that it changes numpy scalars to Python scalars:
 
-    >>> a = np.array([1, 2])
-    >>> list(a)
+    >>> a = np.uint32([1, 2])
+    >>> a_list = list(a)
+    >>> a_list
     [1, 2]
-    >>> a.tolist()
+    >>> type(a_list)
+    <class 'list'> 
```

Comment:
I'd probably skip this one and the one below, it's not particularly interesting

Reply:
Fine to leave if no one else agrees with me

Reply:
I agree and will change it